com.stencyl.Game.json

```diff
@@ -20,6 +20,10 @@
             "sources": [
                 {
                     "type": "file",
+                    "path": "com.stencyl.Game.png"
```

Comment:
Maybe name it differently to avoid confusion in future about the reasons why the icon is actually an empty file. Maybe something like `placeholder.png`?